pkg/parquetquery/predicate_test.go

```diff
@@ -18,7 +18,7 @@ func TestSubstringPredicate(t *testing.T) {
 		keptChunks: 1,
 		keptPages:  1,
 		keptValues: 2,
-		writeData: func(w *parquet.Writer) {
+		writeData: func(w *parquet.Writer) { //nolint:all
```

Comment:
what does the linter not like about this?

Reply:
Oops. Forgot to put the reason there. But parquet.Writer is deprecated